packages/playground/tests/utils/isValidSmtp.test.ts

```diff
@@ -0,0 +1,24 @@
+import { describe, expect, it } from "vitest";
+
+import { isValidSmtp } from "../../src/utils/validators";
+
+describe("isValidSmtp", () => {
```

Comment:
Probably need more cases e.g empty string, multiple invalid forms of email as well, usage of special characters

Reply:
as the function is deleted, the test case is deleted too.